client/src/commons/DatePick/DateRangePick.tsx

```diff
@@ -6,13 +6,14 @@ import useStyles from './DateRangePickStyles';
 
 const DateRangePick: React.FC<Props> = (props: Props): JSX.Element => {
 
-    const { startDate, onStartDateChange, endDate, onEndDateChange, minDate, maxDate } = props;
+    const { id, startDate, onStartDateChange, endDate, onEndDateChange, minDate, maxDate } = props;
     const classes = useStyles();
 
     return (
-        <Grid container alignItems='center' xs={12} spacing={1}>
+        <Grid item container alignItems='center' xs={12} spacing={1} id={id}>
```

Comment:
maybe check if id is not undefined before passing it to grid

Reply:
![image](https://user-images.githubusercontent.com/68457306/107655189-fdd85800-6c8b-11eb-92bd-df324c6291ad.png)
when the id is undefined it doesn't render it